doc/content/verification_and_validation/ver-1kc-2.md

```diff
@@ -41,44 +77,55 @@ where $R$ is the ideal gas constant in J/mol/K, $T$ is the temperature in K, $K$
 
 ## Results
 
-We assume that $K = \frac{10}{\sqrt{RT}}$, which is expected to result in $C_1 = 10 \sqrt{C_2}$ at equilibrium.
-As illustrated in [ver-1kc-2_comparison_time_k10], similarly to ver-1kc-1, T$_2$ and H$_2$ pressures reach equilibrium in both enclosures. What is new, however, is that HT is produced in both enclosures following the sorption law. The concentration ratios for T$_2$, H$_2$, and HT between enclosures 1 and 2, shown in [ver-1kc-2_concentration_ratio_T2_k10], [ver-1kc-2_concentration_ratio_H2_k10], and [ver-1kc-2_concentration_ratio_HT_k10], demonstrate that the results obtained with TMAP8 are consistent with the analytical results derived from the sorption law for $K \sqrt{RT} = 10$.
+We assume that $K = 10/\sqrt{RT}$, which is expected to result in $C_1 = 10 \sqrt{C_2}$ at equilibrium.
+As illustrated in [ver-1kc-2_comparison_time_k10], similarly to ver-1kc-1, T$_2$ and H$_2$ pressures reach equilibrium in both enclosures. What is new, however, is that HT is produced in both enclosures following the sorption law.
+
+Thus, it is crucial to ensure that the chemical equilibrium between HT, T$_2$ and H$_2$ is achieved. This can be verified in both enclosures by examining the ratio between $P_{\text{HT}}$ and $\sqrt{P_{\text{H}_2} P_{\text{T}_2}}$, which must equal $\eta=2$.
+As shown in [ver-1kc-2_equilibrium_constant_k10], this ratio approaches $\eta=2$ for both enclosures, as observed in TMAP7. However, achieving this balance involves a compromise. On one hand, $K_1$ must be sufficiently large to ensure that the chemical kinetics in Enclosure 1 are significantly faster than other processes, such as diffusion and surface sorption. On the other hand, $K_2$ should not be excessively large, as this could hinder the diffusion of species into Enclosure 2, where no species are initially present.
```

Comment:
```suggestion
As shown in [ver-1kc-2_equilibrium_constant_k10], this ratio approaches $\eta=2$ for both enclosures, as observed in TMAP7. However, achieving this balance involves a compromise to reproduce TMAP7's results. The ratio of $K_1$ and $K_2$ must respect [eq:eta]. The values of $K_1$ and $K_2$ must also be large enough to ensure that the kinetics of chemical reactions are faster than diffusion or surface permeation to be closer to the equilibrium assumption used in TMAP7. Moreover, using very high values for $K_1$ and $K_2$ would lead to an unnecessary increase in computational costs. 
```

Reply:
Initially, there is no H2, T2, or HT in enclosure 2. If $K_1$ and $K_2$, which govern the chemical reaction kinetics, become too large compared to the diffusion rate in enclosure 2, chemical reactions will occur while no species will have sufficient time to diffuse into enclosure 2. Consequently, I believe that the equilibrium $\eta = \sqrt{2K_1/K_2}$ will not be maintained in enclosure 2. That's why I think that $K_1$ and $K_2$ must be sufficiently high to respect $\eta = 2$ in enclosure 1, but not so large in enclosure 2. 

Does this seem correct to you?